testing/python/invocation_scope.py

```diff
@@ -0,0 +1,138 @@
+
+
+class TestAcceptance:
+    """
+    Complete acceptance test for a invocation-scoped fixture.
+    """
+
```

Comment:
i think there should be a very simple test case with just two fixtures, e.g. a session-scoped and a module-scoped one requesting an invocation scoped one.